content/docs/guides/auth.md

```diff
@@ -505,8 +579,17 @@ var stub = new helloworld.Greeter('localhost:50051', grpc.credentials.createInse
 ##### With server authentication SSL/TLS
 
 ```js
-var ssl_creds = grpc.credentials.createSsl(root_certs);
-var stub = new helloworld.Greeter('myservice.example.com', ssl_creds);
+const root_cert = fs.readFileSync('path/to/root-cert');
+const ssl_creds = grpc.credentials.createSsl(root_cert);
+const stub = new helloworld.Greeter('myservice.example.com', ssl_creds);
+```
+
+##### With server authentication SSL/TLS and disabled hostname checking
```

Comment:
I don't see "disabled hostname checking" examples for the other languages, and I don't think we should add it. That option is intended only for testing and I would want to at least make that clearer if we are going to recommend it here.

Reply:
Done